assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -183,6 +187,40 @@ def _check_and_format_azure_endpoint_url(
     return url
 
 
+def get_aad_credential():
+    """Get AzureMLOnBehalfOfCredential."""
+    return AzureMLOnBehalfOfCredential(
+        AZUREML_SYNAPSE_CLUSTER_IDENTIFIER=os.environ[
+            "AZUREML_SYNAPSE_CLUSTER_IDENTIFIER"
+        ],
+        AZUREML_SYNAPSE_TOKEN_SERVICE_ENDPOINT=os.environ[
+            "AZUREML_SYNAPSE_TOKEN_SERVICE_ENDPOINT"
+        ],
+        AZUREML_RUN_ID=os.environ["AZUREML_RUN_ID"],
+        AZUREML_RUN_TOKEN_EXPIRY=os.environ["AZUREML_RUN_TOKEN_EXPIRY"],
+    )
+
+
+def get_ml_client(connection_name, subscription_id, resource_group_name, workspace_name, use_aad=False):
+    """Get MLClient handler with necessary authentication."""
+    get_aad_credential()
```

Comment:
Why call this here if we don't use the value?

Reply:
Yeah we never used it before too, I just moved it to a helper method.  I'm afraid to remove it though because it used to exist there before for some reason and I don't know what it is doing, maybe it will cause the auth to fail in some scenarios.

Reply:
Okay sounds good. I'm not familiar enough with what it does to recommend removing it either so we can leave it